cherrypy/test/test_states.py

```diff
@@ -161,9 +161,14 @@ def test_1_Restart(self):
         self.assertEqual(db_connection.running, False)
         self.assertEqual(len(db_connection.threads), 0)
 
+    # @pytest.mark.skipif(os.name == 'nt', 'see pytest-dev/pytest-forked#44')
+    @pytest.mark.forked
+    @pytest.mark.xfail(bool(os.getenv('CI')), reason='KeyboardInterrupt #1873')
```

Comment:
This should probably be unconditional. There's no reason to mark it as failed in local runs if it's known that it's expected to fail.